src/test/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/CreateSDORespondent1NotificationHandlerTest.java

```diff
@@ -105,6 +114,41 @@ void shouldNotifyRespondentLiP_whenInvoked() {
                     CLAIM_REFERENCE_NUMBER, LEGACY_CASE_REFERENCE,
                     CLAIM_LEGAL_ORG_NAME_SPEC, caseData.getRespondent1().getPartyName()
                 ),
+                legacyCaseReference
+            );
+        }
+
+        @Test
+        void shouldNotifyRespondentLiPWithBilingual_whenInvoked() {
+            Party party = PartyBuilder.builder()
+                .individual(defendantName)
+                .partyEmail(defendantEmail)
+                .build();
+            CaseData caseData = CaseDataBuilder.builder().atStateClaimDetailsNotified().build()
+                .toBuilder()
+                .respondent1Represented(YesOrNo.NO)
+                .caseDataLiP(CaseDataLiP.builder()
+                                 .respondent1LiPResponse(RespondentLiPResponse.builder()
+                                                             .respondent1ResponseLanguage("BOTH").build()).build())
+                .respondent1(party)
+                .build();
+            CallbackParams params = CallbackParams.builder()
+                .caseData(caseData)
+                .type(ABOUT_TO_SUBMIT)
+                .request(CallbackRequest.builder()
+                             .eventId(CaseEvent.NOTIFY_RESPONDENT_SOLICITOR1_SDO_TRIGGERED.name())
+                             .build())
+                .build();
+
+            handler.handle(params);
+
+            verify(notificationService).sendMail(
+                caseData.getRespondent1().getPartyEmail(),
+                "template-id",
```

Comment:
could use constants instead of just stirngs to compare - private static final String

Reply:
converted to constants